src/AdherentProfile/AdherentProfile.php

```diff
@@ -39,6 +39,7 @@ class AdherentProfile implements MembershipInterface
      * @Assert\Length(
      *     min=2,
      *     max=50,
+     *     allowEmptyString=true,
```

Comment:
Why add this, if the field has `@Assert\NotBlank`?

Reply:
![image](https://user-images.githubusercontent.com/4582866/101382517-59671d80-38b8-11eb-999f-28d4011e24f6.png)
because it creates another error in addition of `NotBlank`